http-gateway/swagger.yaml

```diff
@@ -49,6 +49,42 @@ paths:
           $ref: '#/components/responses/badRequest'
         401:
           $ref: '#/components/responses/unauthorized'
+    delete:
+      tags:
+        - 'Devices'
+      summary: 'Delete owned devices'
```

Comment:
you can skip word owned. It might raise the question "what is owned device", "if there can be connected not owned device", and so on. Each device connected to the plgd cloud is owned.